graphql/introspection/introspection.go

```diff
@@ -444,3 +444,17 @@ func RunIntrospectionQuery(schema *graphql.Schema) ([]byte, error) {
 
 	return json.MarshalIndent(value, "", "  ")
 }
+
+func IntrospectionQueryTypeOrSelection(typeName, selectionName string) bool {
+	if selectionName == "__schema" ||
```

Comment:
This function is not used anywhere?

Reply:
my plan is to use this helper in our backend repo - I can move it there directly.